nuitka/build/SconsCompilerSettings.py

```diff
@@ -787,7 +787,7 @@ def setupCCompiler(env, lto_mode, pgo_mode, job_count, onefile_compile):
 
     # Even if console is forced, for Win32 it means to specify Windows
     # subsystem, we can still attach or create.
-    if env.console_mode != "force":
+    if env.console_mode not in ["force", "hide"]:
```

Comment:
Probably should list the modes that actually are not console, but actually want Windows subsystem instead. Also using a tuple over a list when doing this operation is something I regard cleaner.